interface/skin.dmf

```diff
@@ -1543,7 +1541,7 @@ window "mainwindow"
 		saved-params = ""
 		is-flat = false
 		text = "Hotkey Toggle"
-		command = ".winset \"mainwindow.macro != mainwindow.macro_hotkey_mode_inactive ? mainwindow.macro=mainwindow.macro_hotkey_mode_inactive hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5 : mainwindow.macro=mainwindow.macro_hotkey_mode_active hotkey_toggle.is-checked=true mapwindow.map.focus=true input.background-color=#f0f0f0\""
+		command = ".winset \"mainwindow.macro != macro ? mainwindow.macro=macro hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5 : mainwindow.macro=hotkeymode hotkey_toggle.is-checked=true mapwindow.map.focus=true input.background-color=#f0f0f0\""

```

Comment:
So, in BYOND you can actually SET window values by just calling <windowname>.<variable> but you can't actually get them using the same method.